clients/src/main/java/org/apache/kafka/common/requests/ApiVersionsResponse.java

```diff
@@ -212,24 +248,32 @@ public static ApiVersionCollection collectApis(Set<ApiKeys> apiKeys) {
      * @param listenerType the listener type which constrains the set of exposed APIs
      * @param minRecordVersion min inter broker magic
      * @param activeControllerApiVersions controller ApiVersions
+     * @param enableUnstableLastVersion whether unstable versions should be advertised or not
```

Comment:
i'm a bit confused on the word "last". does this mean we enable only if the last version is unstable? my understanding is we enable all unstable versions or we don't

Reply:
that's right but only the last version of an api could be unstable. this is where the "last" comes from.

Reply:
Do we have a place where all this new unstable api usage is documented? It could be a bit confusing for folks who didn't take a look at this PR 😅